src/Adapters/Email/MailAdapter.js

```diff
@@ -0,0 +1,6 @@
+export class MailAdapter {
+  sendVerificationEmail(options) {}
+  sendMail(options) {}
```

Comment:
the sendVerificationEmail is an opportunity for the adapter to provide a template, HTML + text etc... 
That's how I see it, then if it's a *mail* adapter, it should be able to send a mail, this way a MailController can safely send email whatever the configured adapter is.